src/components/AppRouter/AppRouter.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { Container } from "react-bootstrap";
+import { Routes, Route } from "react-router-dom";
+import { authRoutes, publicRoutes } from "../../routes";
+import { isAuth } from "../../data/Data";
+
+const AppRouter = () => {
+  return (
+    <Container style={{ padding: "40px" }}>
+      <Routes>
+        {isAuth &&
```

Comment:
Really don't like this idea. take a look at High Order Components design. Or check alternatives how to organize routing